fauna/fauna-shell
# fix(OSS-878): Fixed settings new alias as a default one

Fixed settings new alias as a default one during cloud-login interactive mode

### Notes
[Jira Ticket](https://faunadb.atlassian.net/browse/OSS-878)

### How to test
1. Run `fauna cloud-login`
2. Add at least 1 endpoints
3. Run `fauna cloud-login` again
4. In the last step you'll be prompted to set the current endpoint as a default. Choose yes.
5. Check if the latest endpoint has been added as a default in the `~/.fauna-shell` file

### Screenshots
![image](https://user-images.githubusercontent.com/3454949/128871070-10441f32-2df4-4c5a-b1be-bb9c3b1fb793.png)
